quetz/rest_models.py

```diff
@@ -65,8 +66,10 @@ class ChannelBase(BaseModel):
     private: bool = Field(True, title="channel should be private")
     size_limit: Optional[int] = Field(None, title="size limit of the channel")
     ttl: int = Field(36000, title="ttl of the channel")
-    mirror_channel_url: Optional[str] = Field(None, pattern="^(http|https)://.+")
-    mirror_mode: Optional[MirrorMode] = Field(None)
+    mirror_channel_url: Optional[Union[URLType, List[URLType]]] = Field(
+        None,
+    )
+    mirror_mode: Optional[MirrorMode] = Field(None, nullable=True)
```

Comment:
Can we make this an enum so that only `proxy` and `mirror` are allowed?

Reply:
Is this not already an enum..?